syntax = "proto3";

service KV {
    
    rpc getKV(Index) returns (KVdata) {}

}

message Index {
    int32 centroidid = 1;
}

message KVdata {
    repeated KeyValue kv = 1;
}

message KeyValue {
    int32 key = 1;
    Point value = 2;
}

message Point {
    double x = 1;
    double y = 2;
}

